specifyweb/frontend/js_src/lib/components/DataModel/__tests__/businessRules.test.ts

```diff
@@ -168,6 +169,64 @@ describe('Collection Object business rules', () => {
     expect(collectionObject.get('catalogNumber')).toBeDefined();
     expect(collectionObject.get('description')).toBeNull();
   });
+
+  test('CollectionObject -> determinations: New determinations are current by default', async () => {
+    const collectionObject = getBaseCollectionObject();
+    const determinations =
+      collectionObject.getDependentResource('determinations');
+
+    determinations?.add(new tables.Determination.Resource());
+
+    // Old determination gets unchecked as current
+    expect(determinations?.models[0].get('isCurrent')).toBe(false);
+    // New determination becomes current by default
+    expect(determinations?.models[1].get('isCurrent')).toBe(true);
```

Comment:
(just a note) 

How reliable is it that the first Determination in the collection `at models[0]` is the one defined on the `baseCollectionObject`? 

I know a comparator for a Collection can be explicitly defined, and BackBone implicitly keeps the Collection sorted when objects are added (see https://github.com/jashkenas/backbone/blob/4e642088c4810938a0510fa89456ef2619f446da/backbone.js#L1032-L1051), but I am not sure of the behavior when a comparator is not defined for a Collection (I assume it sorts by their internal `cid`, or when the resources/models are added to the Collection?) 

If there is a consistent behavior, we can keep this as is! 
Otherwise, we can probably include some identifier on each of the Determinations and filter them from the Collection when needed (this would also handle the case when the BaseCollectionObject's determinations are modified) .

Reply:
Looks like it gets sorted by its `cid`